include/quill/core/Common.h

```diff
@@ -30,8 +30,8 @@ namespace detail
 /**
  * Cache line size
  */
-static constexpr size_t CACHE_LINE_SIZE{64u};
-static constexpr size_t CACHE_LINE_ALIGNED{2 * CACHE_LINE_SIZE};
+static constexpr size_t cache_line_size{64u};
+static constexpr size_t cache_line_aligned{2 * cache_line_size};
```

Comment:
lets please rename to QUILL_CACHE_LINE_SIZE and QUILL_CACHE_LINE_ALIGNED to maintain consistency with the rest of the repo